iotdb/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java

```diff
@@ -717,43 +792,10 @@ private TSExecuteStatementResp executeUpdateStatement(String statement)
           "Statement is a query statement.");
     }
 
-    // if operation belongs to add/delete/update
```

Comment:
pls reserve this comment... and it says if the operation is add/delete/update, why you name the function as `executeUpdateStatement(physicalPlan);`

Reply:
ok, add this comment already. I change those codes in `executeUpdateStatement(statement)` to `executeUpdateStatement(physicalPlan)` because these code are exactly same and duplicate.  Name  `executeUpdateStatement ` is a historical legacy of naming, not something I have changed.  `executeUpdateStatement` process non-query operation， including add/delete/delete.